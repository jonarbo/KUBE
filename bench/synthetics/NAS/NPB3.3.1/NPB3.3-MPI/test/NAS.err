Unloading module gcc version 4.6.0
Initial gcc version: 4.6.0
Current gcc version: 4.4.4
Unloading module mpi-openmpi version 1.4.3-gcc-4.6.0
Current mpi-openmpi version: none
Loading module gcc version 4.6.0
Initial gcc version: 4.4.4
Current gcc version: 4.6.0
Loading module mpi-openmpi version 1.4.3-gcc-4.6.0
Current mpi-openmpi version: 1.4.3
