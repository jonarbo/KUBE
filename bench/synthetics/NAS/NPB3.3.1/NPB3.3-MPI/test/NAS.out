

 NAS Parallel Benchmarks 3.3 -- FT Benchmark

 No input file inputft.data. Using compiled defaults
 Size                :  256x 256x 128
 Iterations          :              6
 Number of processes :              8
 Processor array     :         1x   8
 Layout type         :             1D
 Initialization time =  0.22505903244018555     
 T =    1     Checksum =    5.046735008193D+02    5.114047905510D+02
 T =    2     Checksum =    5.059412319734D+02    5.098809666433D+02
 T =    3     Checksum =    5.069376896287D+02    5.098144042213D+02
 T =    4     Checksum =    5.077892868474D+02    5.101336130759D+02
 T =    5     Checksum =    5.085233095391D+02    5.104914655194D+02
 T =    6     Checksum =    5.091487099959D+02    5.107917842803D+02
 Result verification successful
 class = A


 FT Benchmark Completed.
 Class           =                        A
 Size            =            256x 256x 128
 Iterations      =                        6
 Time in seconds =                     1.18
 Total processes =                        8
 Compiled procs  =                        8
 Mop/s total     =                  6032.00
 Mop/s/process   =                   754.00
 Operation type  =           floating point
 Verification    =               SUCCESSFUL
 Version         =                    3.3.1
 Compile date    =              12 Sep 2012

 Compile options:
    MPIF77       = mpif77
    FLINK        = $(MPIF77)
    FMPI_LIB     = -L/usr/local/lib -lmpi
    FMPI_INC     = -I/usr/local/include
    FFLAGS       = -O
    FLINKFLAGS   = -O
    RAND         = randi8


 Please send feedbacks and/or the results of this run to:

 NPB Development Team 
 Internet: npb@nas.nasa.gov


Job  /apps/admin/lsf/7.0/linux2.6-glibc2.3-x86_64/bin/openmpi_wrapper ../bin/ft.A.8

TID   HOST_NAME   COMMAND_LINE            STATUS            TERMINATION_TIME
===== ========== ================  =======================  ===================
00000 ci078-ib   ../bin/ft.A.8     Done                     09/12/2012 15:59:00
00001 ci078-ib   ../bin/ft.A.8     Done                     09/12/2012 15:59:00
00002 ci078-ib   ../bin/ft.A.8     Done                     09/12/2012 15:59:00
00003 ci078-ib   ../bin/ft.A.8     Done                     09/12/2012 15:59:00
00004 ci078-ib   ../bin/ft.A.8     Done                     09/12/2012 15:59:00
00005 ci078-ib   ../bin/ft.A.8     Done                     09/12/2012 15:59:00
00006 ci078-ib   ../bin/ft.A.8     Done                     09/12/2012 15:59:00
00007 ci078-ib   ../bin/ft.A.8     Done                     09/12/2012 15:59:00


 NAS Parallel Benchmarks 3.3 -- MG Benchmark

 No input file. Using compiled defaults 
 Size:  256x 256x 256  (class A)
 Iterations:    4
 Number of processes:      8

 Initialization time:           0.241 seconds

  iter    1
  iter    4

 Benchmark completed 
 VERIFICATION SUCCESSFUL 
 L2 Norm is  0.2433365309069E-05
 Error is    0.1992821877158E-12


 MG Benchmark Completed.
 Class           =                        A
 Size            =            256x 256x 256
 Iterations      =                        4
 Time in seconds =                     0.48
 Total processes =                        8
 Compiled procs  =                        8
 Mop/s total     =                  8134.49
 Mop/s/process   =                  1016.81
 Operation type  =           floating point
 Verification    =               SUCCESSFUL
 Version         =                    3.3.1
 Compile date    =              12 Sep 2012

 Compile options:
    MPIF77       = mpif77
    FLINK        = $(MPIF77)
    FMPI_LIB     = -L/usr/local/lib -lmpi
    FMPI_INC     = -I/usr/local/include
    FFLAGS       = -O
    FLINKFLAGS   = -O
    RAND         = randi8


 Please send feedbacks and/or the results of this run to:

 NPB Development Team 
 Internet: npb@nas.nasa.gov


Job  /apps/admin/lsf/7.0/linux2.6-glibc2.3-x86_64/bin/openmpi_wrapper ../bin/mg.A.8

TID   HOST_NAME   COMMAND_LINE            STATUS            TERMINATION_TIME
===== ========== ================  =======================  ===================
00000 ci078-ib   ../bin/mg.A.8     Done                     09/12/2012 15:59:05
00001 ci078-ib   ../bin/mg.A.8     Done                     09/12/2012 15:59:05
00002 ci078-ib   ../bin/mg.A.8     Done                     09/12/2012 15:59:05
00003 ci078-ib   ../bin/mg.A.8     Done                     09/12/2012 15:59:05
00004 ci078-ib   ../bin/mg.A.8     Done                     09/12/2012 15:59:05
00005 ci078-ib   ../bin/mg.A.8     Done                     09/12/2012 15:59:05
00006 ci078-ib   ../bin/mg.A.8     Done                     09/12/2012 15:59:05
00007 ci078-ib   ../bin/mg.A.8     Done                     09/12/2012 15:59:05


 NAS Parallel Benchmarks 3.3 -- LU Benchmark

 Size:   64x  64x  64
 Iterations:  250
 Number of processes:     8

 Time step    1
 Time step   20
 Time step   40
 Time step   60
 Time step   80
 Time step  100
 Time step  120
 Time step  140
 Time step  160
 Time step  180
 Time step  200
 Time step  220
 Time step  240
 Time step  250

 Verification being performed for class A
 Accuracy setting for epsilon =  0.1000000000000E-07
 Comparison of RMS-norms of residual
           1   0.7790210760669E+03 0.7790210760669E+03 0.1400980892119E-13
           2   0.6340276525969E+02 0.6340276525969E+02 0.5715473035811E-14
           3   0.1949924972729E+03 0.1949924972729E+03 0.8745477850039E-14
           4   0.1784530116042E+03 0.1784530116042E+03 0.3185343769198E-15
           5   0.1838476034946E+04 0.1838476034946E+04 0.1137810759735E-13
 Comparison of RMS-norms of solution error
           1   0.2996408568547E+02 0.2996408568547E+02 0.1185657295234E-14
           2   0.2819457636500E+01 0.2819457636500E+01 0.1370326007271E-13
           3   0.7347341269878E+01 0.7347341269877E+01 0.7373944071964E-14
           4   0.6713922568778E+01 0.6713922568778E+01 0.9260233305063E-15
           5   0.7071531568839E+02 0.7071531568839E+02 0.1185656063379E-13
 Comparison of surface integral
               0.2603092560489E+02 0.2603092560489E+02 0.1364804975715E-15
 Verification Successful


 LU Benchmark Completed.
 Class           =                        A
 Size            =             64x  64x  64
 Iterations      =                      250
 Time in seconds =                     9.22
 Total processes =                        8
 Compiled procs  =                        8
 Mop/s total     =                 12934.52
 Mop/s/process   =                  1616.81
 Operation type  =           floating point
 Verification    =               SUCCESSFUL
 Version         =                    3.3.1
 Compile date    =              12 Sep 2012

 Compile options:
    MPIF77       = mpif77
    FLINK        = $(MPIF77)
    FMPI_LIB     = -L/usr/local/lib -lmpi
    FMPI_INC     = -I/usr/local/include
    FFLAGS       = -O
    FLINKFLAGS   = -O
    RAND         = (none)


 Please send feedbacks and/or the results of this run to:

 NPB Development Team 
 Internet: npb@nas.nasa.gov


Job  /apps/admin/lsf/7.0/linux2.6-glibc2.3-x86_64/bin/openmpi_wrapper ../bin/lu.A.8

TID   HOST_NAME   COMMAND_LINE            STATUS            TERMINATION_TIME
===== ========== ================  =======================  ===================
00000 ci078-ib   ../bin/lu.A.8     Done                     09/12/2012 15:59:19
00001 ci078-ib   ../bin/lu.A.8     Done                     09/12/2012 15:59:19
00002 ci078-ib   ../bin/lu.A.8     Done                     09/12/2012 15:59:19
00003 ci078-ib   ../bin/lu.A.8     Done                     09/12/2012 15:59:19
00004 ci078-ib   ../bin/lu.A.8     Done                     09/12/2012 15:59:19
00005 ci078-ib   ../bin/lu.A.8     Done                     09/12/2012 15:59:19
00006 ci078-ib   ../bin/lu.A.8     Done                     09/12/2012 15:59:19
00007 ci078-ib   ../bin/lu.A.8     Done                     09/12/2012 15:59:19


 NAS Parallel Benchmarks 3.3 -- IS Benchmark

 Size:  8388608  (class A)
 Iterations:   10
 Number of processes:     8

   iteration
        1
        2
        3
        4
        5
        6
        7
        8
        9
        10


 IS Benchmark Completed
 Class           =                        A
 Size            =                  8388608
 Iterations      =                       10
 Time in seconds =                     0.16
 Total processes =                        8
 Compiled procs  =                        8
 Mop/s total     =                   516.69
 Mop/s/process   =                    64.59
 Operation type  =              keys ranked
 Verification    =               SUCCESSFUL
 Version         =                    3.3.1
 Compile date    =              12 Sep 2012

 Compile options:
    MPICC        = mpicc
    CLINK        = $(MPICC)
    CMPI_LIB     = # -L/usr/local/lib -lmpi
    CMPI_INC     = -I/usr/local/include
    CFLAGS       = -O2
    CLINKFLAGS   = -O2


 Please send feedbacks and/or the results of this run to:

 NPB Development Team
 npb@nas.nasa.gov


Job  /apps/admin/lsf/7.0/linux2.6-glibc2.3-x86_64/bin/openmpi_wrapper ../bin/is.A.8

TID   HOST_NAME   COMMAND_LINE            STATUS            TERMINATION_TIME
===== ========== ================  =======================  ===================
00000 ci078-ib   ../bin/is.A.8     Done                     09/12/2012 15:59:24
00001 ci078-ib   ../bin/is.A.8     Done                     09/12/2012 15:59:24
00002 ci078-ib   ../bin/is.A.8     Done                     09/12/2012 15:59:24
00003 ci078-ib   ../bin/is.A.8     Done                     09/12/2012 15:59:24
00004 ci078-ib   ../bin/is.A.8     Done                     09/12/2012 15:59:24
00005 ci078-ib   ../bin/is.A.8     Done                     09/12/2012 15:59:24
00006 ci078-ib   ../bin/is.A.8     Done                     09/12/2012 15:59:24
00007 ci078-ib   ../bin/is.A.8     Done                     09/12/2012 15:59:24

 NAS Parallel Benchmarks 3.3 -- EP Benchmark

 Number of random numbers generated:       536870912
 Number of active processes:                       8

EP Benchmark Results:

CPU Time =    2.6068
N = 2^   28
No. Gaussian Pairs =     210832767.
Sums =    -4.295875165636595D+03   -1.580732573678463D+04
Counts:
  0      98257395.
  1      93827014.
  2      17611549.
  3       1110028.
  4         26536.
  5           245.
  6             0.
  7             0.
  8             0.
  9             0.


 EP Benchmark Completed.
 Class           =                        A
 Size            =                536870912
 Iterations      =                        0
 Time in seconds =                     2.61
 Total processes =                        8
 Compiled procs  =                        8
 Mop/s total     =                   205.95
 Mop/s/process   =                    25.74
 Operation type  = Random numbers generated
 Verification    =               SUCCESSFUL
 Version         =                    3.3.1
 Compile date    =              12 Sep 2012

 Compile options:
    MPIF77       = mpif77
    FLINK        = $(MPIF77)
    FMPI_LIB     = -L/usr/local/lib -lmpi
    FMPI_INC     = -I/usr/local/include
    FFLAGS       = -O
    FLINKFLAGS   = -O
    RAND         = randi8


 Please send feedbacks and/or the results of this run to:

 NPB Development Team 
 Internet: npb@nas.nasa.gov


Job  /apps/admin/lsf/7.0/linux2.6-glibc2.3-x86_64/bin/openmpi_wrapper ../bin/ep.A.8

TID   HOST_NAME   COMMAND_LINE            STATUS            TERMINATION_TIME
===== ========== ================  =======================  ===================
00000 ci078-ib   ../bin/ep.A.8     Done                     09/12/2012 15:59:31
00001 ci078-ib   ../bin/ep.A.8     Done                     09/12/2012 15:59:31
00002 ci078-ib   ../bin/ep.A.8     Done                     09/12/2012 15:59:31
00003 ci078-ib   ../bin/ep.A.8     Done                     09/12/2012 15:59:31
00004 ci078-ib   ../bin/ep.A.8     Done                     09/12/2012 15:59:31
00005 ci078-ib   ../bin/ep.A.8     Done                     09/12/2012 15:59:31
00006 ci078-ib   ../bin/ep.A.8     Done                     09/12/2012 15:59:31
00007 ci078-ib   ../bin/ep.A.8     Done                     09/12/2012 15:59:31


 NAS Parallel Benchmarks 3.3 -- CG Benchmark

 Size:      14000
 Iterations:    15
 Number of active processes:     8
 Number of nonzeroes per row:       11
 Eigenvalue shift: .200E+02

   iteration           ||r||                 zeta
        1       0.26677899255571E-12    19.9997581277040
        2       0.26402893870124E-14    17.1140495745506
        3       0.26364629967417E-14    17.1296668946143
        4       0.26233845216830E-14    17.1302113581193
        5       0.26766729195432E-14    17.1302338856353
        6       0.26341234467333E-14    17.1302349879482
        7       0.26018681594959E-14    17.1302350498916
        8       0.26076001869473E-14    17.1302350537510
        9       0.25882472324258E-14    17.1302350540101
       10       0.25710771747069E-14    17.1302350540284
       11       0.25556814995438E-14    17.1302350540298
       12       0.25867780925789E-14    17.1302350540299
       13       0.25299331252052E-14    17.1302350540299
       14       0.25376913365533E-14    17.1302350540299
       15       0.24928830494495E-14    17.1302350540299
 Benchmark completed 
 VERIFICATION SUCCESSFUL 
 Zeta is     0.1713023505403E+02
 Error is    0.5247076633225E-13


 CG Benchmark Completed.
 Class           =                        A
 Size            =                    14000
 Iterations      =                       15
 Time in seconds =                     0.31
 Total processes =                        8
 Compiled procs  =                        8
 Mop/s total     =                  4790.13
 Mop/s/process   =                   598.77
 Operation type  =           floating point
 Verification    =               SUCCESSFUL
 Version         =                    3.3.1
 Compile date    =              12 Sep 2012

 Compile options:
    MPIF77       = mpif77
    FLINK        = $(MPIF77)
    FMPI_LIB     = -L/usr/local/lib -lmpi
    FMPI_INC     = -I/usr/local/include
    FFLAGS       = -O
    FLINKFLAGS   = -O
    RAND         = randi8


 Please send feedbacks and/or the results of this run to:

 NPB Development Team 
 Internet: npb@nas.nasa.gov


Job  /apps/admin/lsf/7.0/linux2.6-glibc2.3-x86_64/bin/openmpi_wrapper ../bin/cg.A.8

TID   HOST_NAME   COMMAND_LINE            STATUS            TERMINATION_TIME
===== ========== ================  =======================  ===================
00000 ci078-ib   ../bin/cg.A.8     Done                     09/12/2012 15:59:36
00001 ci078-ib   ../bin/cg.A.8     Done                     09/12/2012 15:59:36
00002 ci078-ib   ../bin/cg.A.8     Done                     09/12/2012 15:59:36
00003 ci078-ib   ../bin/cg.A.8     Done                     09/12/2012 15:59:36
00004 ci078-ib   ../bin/cg.A.8     Done                     09/12/2012 15:59:36
00005 ci078-ib   ../bin/cg.A.8     Done                     09/12/2012 15:59:36
00006 ci078-ib   ../bin/cg.A.8     Done                     09/12/2012 15:59:36
00007 ci078-ib   ../bin/cg.A.8     Done                     09/12/2012 15:59:36

------------------------------------------------------------
Sender: LSF System <lsfadmin@ci078-ib>
Subject: Job 638796: <#!/bin/bash -l; #BSUB -o NAS.out;#BSUB -e NAS.err;#BSUB -q rh6_q2hr ;#BSUB -W 20 ;#BSUB -n 8;#BSUB -a openmpi;#BSUB -R span[ptile=8]; module purge;module load gcc mpi-openmpi; mpirun.lsf ../bin/ft.A.8;mpirun.lsf ../bin/mg.A.8;mpirun.lsf ../bin/lu.A.8;mpirun.lsf ../bin/is.A.8;mpirun.lsf ../bin/ep.A.8;mpirun.lsf ../bin/cg.A.8> Done

Job <#!/bin/bash -l; #BSUB -o NAS.out;#BSUB -e NAS.err;#BSUB -q rh6_q2hr ;#BSUB -W 20 ;#BSUB -n 8;#BSUB -a openmpi;#BSUB -R span[ptile=8]; module purge;module load gcc mpi-openmpi; mpirun.lsf ../bin/ft.A.8;mpirun.lsf ../bin/mg.A.8;mpirun.lsf ../bin/lu.A.8;mpirun.lsf ../bin/is.A.8;mpirun.lsf ../bin/ep.A.8;mpirun.lsf ../bin/cg.A.8> was submitted from host <ct001-ib> by user <naranjja> in cluster <Noor>.
Job was executed on host(s) <8*ci078-ib>, in queue <rh6_q2hr>, as user <naranjja> in cluster <Noor>.
</home/naranjja> was used as the home directory.
</home/naranjja/Devel/KaBS/bench/synthetics/NAS/NPB3.3.1/NPB3.3-MPI/test> was used as the working directory.
Started at Wed Sep 12 15:58:56 2012
Results reported at Wed Sep 12 15:59:39 2012

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
#!/bin/bash -l

#BSUB -o NAS.out
#BSUB -e NAS.err
#BSUB -q rh6_q2hr 
#BSUB -W 20 
#BSUB -n 8
#BSUB -a openmpi
#BSUB -R span[ptile=8]

module purge
module load gcc mpi-openmpi

mpirun.lsf ../bin/ft.A.8
mpirun.lsf ../bin/mg.A.8
mpirun.lsf ../bin/lu.A.8
mpirun.lsf ../bin/is.A.8
mpirun.lsf ../bin/ep.A.8
mpirun.lsf ../bin/cg.A.8


------------------------------------------------------------

Successfully completed.

Resource usage summary:

    CPU time   :    168.93 sec.
    Max Processes  :         4
    Max Threads    :         5

The output (if any) is above this job summary.



PS:

Read file <NAS.err> for stderr output of this job.

